swagger/v3/api_spec.json

```diff
@@ -161,12 +161,12 @@
         }
       }
     },
-    "/api/v3/partnerships/ecf": {
```

Comment:
in v3 unfortunately we need to amend this manually otherwise it messes with the order